### ISSUE TITLE
Host validation follow up

### ISSUE THREAD

**Issue by whyman** [gerbera/gerbera#2376]
@whyman the fix is working but I have some complains.

UPDATE: Docker image from Master branch

1. The hostname validation prints the warning trace but it's not blocking the requests to the UI (all requests are working no matter the hostname). Is that expected?
```
2022-01-09 13:07:01 warning: Rejected attempt to load host 'gerbera.home' as it does not match configured virtualURL: ''. See https://docs.gerbera.io/en/stable/config-server.html#virtualurl
```

2. Some devices in my network use the IP instead of the hostname. I think you should allow all IP:PORT requests:
```
2022-01-09 13:13:01 warning: Rejected attempt to load host '192.168.1.191:49494' as it does not match configured virtualURL: 'http://gerbera.home/'. See https://docs.gerbera.io/en/stable/config-server.html#virtualurl
```

3. I configured the virtualURL [as in the example](https://docs.gerbera.io/en/stable/config-server.html#virtualurl) but I see `//` at the end of the URL. I can remove the slash from the config but I think It should be done by you when you load the config to make it more user-friendly.
```
2022-01-09 13:12:59   info: The Web UI can be reached by following this link: http://gerbera.home//
```

_Originally posted by @ngosang in https://github.com/gerbera/gerbera/issues/2326#issuecomment-1008287237_

**Comment by whyman** [comment]
1. Is due to a bug in libupnp: https://github.com/pupnp/pupnp/pull/355
2. Is fixed in #2378 
@ngosang

**Comment by ngosang** [comment]
@whyman I confirm 2) is working as expected.

**Comment by KarlStraussberger** [comment]
Implemented in UPNP 1.14.14.
